addons/project/static/src/scss/project_widgets.scss

```diff
@@ -74,6 +74,27 @@
             }
         }
     }
+    @for $size from 2 through length($o-colors) {
+        // Note: the first color is supposed to be invisible so it's ignored
+        .oe_kanban_color_#{$size - 1} {
+            border-left-color: nth($o-colors, $size);
+            &:after {
+                background-color: nth($o-colors, $size);
+            }
+        }
+    }
+    .oe_kanban_color_20 {
+        border-left-color: $o-success;
+        &:after {
+            background-color: $o-success;
+        }
+    }
+    .oe_kanban_color_23 {
+        border-left-color: $o-danger;
+        &:after {
+            background-color: $o-danger;
+        }
+    }
```

Comment:
I would have kept 21 and 22 so that we can use it, just as it's done here :  https://github.com/odoo/odoo/blob/saas-14.5/addons%2Fproject%2Fstatic%2Fsrc%2Fscss%2Fproject_widgets.scss#L11-L22

Thanks a lot !

Reply:
Done